functions/imagemagick/index.js

```diff
@@ -21,9 +21,9 @@ const fs = require('fs');
 const {promisify} = require('util');
 const path = require('path');
 const {Storage} = require('@google-cloud/storage');
-const storage = new Storage();
-const vision = require('@google-cloud/vision').v1p1beta1;
+const vision = require('@google-cloud/vision');
 
+const storage = new Storage();
```

Comment:
Not sure why separated defining and instantiating `Storage`. I'd undo change L24 to L26.